Realm/ObjectServerTests/SwiftObjectServerTests.swift

```diff
@@ -1994,6 +1996,950 @@ extension SwiftObjectServerTests {
         }
         wait(for: [watchEx1, watchEx2], timeout: 60.0)
     }
+
+    // MARK: - Combine promises
+
+    func testEmailPasswordAuthentication() {
```

Comment:
the test names might need a defined prefix or suffix to indicate they are for combine, I see on some other tests they contain `promise` in the name. maybe this is something we need to discuss

Reply:
They should be suffixed with `Combine` as we do for `Watch`.